pyproject.toml

```diff
@@ -36,6 +36,7 @@ azure-keyvault-secrets = { version = "^4.8.0", optional = true }
 boto3 = { version = "^1.19.5", optional = true }
 hvac = { version = ">=0.11.0, <1.1.0", optional = true }
 python-tss-sdk = {version = "~1.2.0", optional = true}
+onepassword-sdk = {version = "^0.1.2", python = ">=3.9", optional = true}
```

Comment:
That version number gives a lot of confidence... :-)

Reply:
Haha! It just came out of Beta... Currently the features that we care about are supported.